static/js/hotkey.js

```diff
@@ -661,7 +679,11 @@ export function process_hotkey(e, hotkey) {
         // Note that there is special handling for Enter/Esc too, but
         // we handle this in other functions.
 
-        if (event_name === "left_arrow" && compose_state.focus_in_empty_compose()) {
+        if (
+            event_name === "left_arrow" &&
+            compose_state.focus_in_empty_compose() &&
+            !check_unread_message()
+        ) {
             message_edit.edit_last_sent_message();
             return true;
         }
```

Comment:
Do we want the logic here to do a `return false` if !check_unread_message?

Reply:
`return true` is when we should handle the process and `return false` is when the browser should, hence I guess `return true` is the right choice along with the feedback.